apps/vs-code-react/src/app/designer/servicesHelper.ts

```diff
@@ -175,23 +175,26 @@ export const getDesignerServices = (
         return Promise.resolve();
       }
 
+      const defaultConfiguration: Record<string, any> = isLocal
+        ? {
+            workflowAppLocation: appSettings.ProjectDirectoryPath,
+          }
+        : {};
+
       const connectionName = connectionId.split('/').splice(-1)[0];
-      const connnectionsInfo = {
+      const connectionsInfo = {
         ...connectionsData?.serviceProviderConnections,
         ...connectionsData?.apiManagementConnections,
       };
-      const connectionInfo = connnectionsInfo[connectionName];
+      const connectionInfo = connectionsInfo[connectionName];
 
       if (connectionInfo) {
         const resolvedConnectionInfo = resolveConnectionsReferences(JSON.stringify(connectionInfo), {}, appSettings);
         delete resolvedConnectionInfo.displayName;
-
-        return {
-          connection: resolvedConnectionInfo,
-        };
+        defaultConfiguration.connection = resolvedConnectionInfo;
       }
 
-      return undefined;
+      return defaultConfiguration;
```

Comment:
We should be always adding workflowAppLocation when it is Local and the dynamicContent property has that configurtion set